src/rachanaGJun20/Array/DuplicateNumberCheck.java

```diff
@@ -0,0 +1,29 @@
+/*Program 2: Find duplicate number on Integer array in Java?
+Hint: There is exactly one number is repeated in the array. You need to write a program to
+find that duplicate number. For example, if an array with length 6 contains numbers {0, 3,
+1, 2, 3}, then duplicated number is 3.*/
+
+package rachanaGJun20.Array;
+
+public class DuplicateNumberCheck{
+
+	// method will find and display duplicate number from given array
+	void findDuplicateNumber(int[] arr) {
+		for (int i = 0; i < arr.length; i++) {
+			for (int j = i + 1; j < arr.length; j++) {
+				if (arr[i] == arr[j]) {
+					System.out.println("Duplicate number in a given array is: " + arr[i]);
+					break;
```

Comment:
Please add else statement to handle scenari Id there is no duplicate element 
Example : arr = { 0, 3, 1, 2}
We wont get any result if no duplicate element found

Reply:
yup got it.... Will update Thanks :)